src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/ValidateApplicantServiceController.java

```diff
@@ -0,0 +1,64 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.exception.FigaroValidationServiceException;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.types.ValidateApplicantServiceRequest;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.types.ValidateApplicantServiceResponse;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+/**
+ * 
+ * LocateMatchingApplicantsController  class. 
+ * 
+ * All response codes are set by FIGARO. 0 = success, all failures will be returned as NEGATIVE values with an associated message.  
+ * 
+ * @author shaunmillargov
+ *
+ */
+@RestController
+public class ValidateApplicantServiceController {
+	
+	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+	
+	@Autowired
+	private FigaroValidationImpl figservice;  // connection to ORDS client. 
+	
+	@RequestMapping(value = "/validateApplicantService",
+			produces = { "application/xml" }, 
+			method = RequestMethod.GET)
+		@ApiOperation(value = "Validate Applicant Service", notes = "", response = ValidateApplicantServiceResponse.class, tags={ "Figaro Validation Services"})
+	  	@ApiResponses(value = {@ApiResponse(code = 200, message = "Successful operation", response = ValidateApplicantServiceResponse.class) })
+	public ValidateApplicantServiceResponse validateApplicantService(
+			@ApiParam(value = "orgPartyId", required = false) @RequestParam(value="orgPartyId", defaultValue="") String orgPartyId,
+			@ApiParam(value = "applPartyId", required = false) @RequestParam(value="applPartyId", defaultValue="") String applPartyId) throws FigaroValidationServiceException {
+		 
+		
+		try {
+			
+			 return figservice.validateApplicantService(
+					
+					new ValidateApplicantServiceRequest( 
+							orgPartyId,
+							applPartyId));
+		 
+		} catch (FigaroValidationServiceException ex) {
+			logger.error("Exception caught as ValidateApplicantService : " + ex.getMessage()); 
+			ex.printStackTrace();
+			return new ValidateApplicantServiceResponse(
+					ex.getMessage(),
+					FigaroValidationServiceConstants.VALIDATION_SERVICE_FAILURE_CD,
+					"F");
```

Comment:
do we want to define this string as well?

Reply:
Good point but the string is specific to this operation and not the entire set of operations within the Figaro Validation Service. I'll create a constant for it. Thx.

Reply:
No problems :)